src/app/tests/suites/certification/Test_TC_DGGEN_2_3.yaml

```diff
@@ -89,3 +89,39 @@ tests:
                 value: 0
       response:
           error: CONSTRAINT_ERROR
+
+    - label:
+          "Step 5: Invoke the PayloadTestRequest command with EnableKey field
+          set to PIXIT.DGGEN.ENABLEKEY, Value field set to 0x1 and Count set to
+          1."
+      runIf: IsExpectedValue
+      PICS: DGGEN.S.C03.Rsp
+      command: "PayloadTestRequest"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: PIXIT.DGGEN.ENABLEKEY
+              - name: "Value"
+                value: 0x1
+              - name: "Count"
+                value: 1
+      response:
+          - values:
+                - name: "Payload"
+                  value: "hex:01"
+
+    - label:
```

Comment:
Please also add an EnableKey != 16 bytes

Reply:
Will do, but that requires test plan spec update first:
https://github.com/CHIP-Specifications/chip-test-plans/pull/3970